esm_runscripts/coupler.py

```diff
@@ -255,20 +255,23 @@ def add_couplings(self, full_config):
                                 if not found_left or not found_right:
                                     sys.exit(0)
 
+                            if "export_mode" in full_config[self.name]:
+                                export_mode = full_config[self.name]["export_mode"]
+                            else:
+                                export_mode = "DEFAULT"
```

Comment:
Can't this whole block be written as `export_mode = full_config[self.name].get("export_mode", DEFAULT)`?

Reply:
I'm sure it can, since it's a hot fix I better try it first :-)

Reply:
tested and pushed